packages/contracts/deploy/new/10_framework/00_ens_registry.ts

```diff
@@ -11,6 +11,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const {network} = hre;
 
+  console.log(`${network.name.toUpperCase()}_DAO_ENS_DOMAIN`, ' kkkkk');
```

Comment:
Write a log message and remove `' kkkkk'`.

Reply:
Oops, sorry. left it there. Removed.